packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -273,6 +296,7 @@ class AnalyzeCommand extends FlutterCommand {
       new RegExp(r'\[lint\] Prefer using lowerCamelCase for constant names.'), // sometimes we have no choice (e.g. when matching other platforms)
       new RegExp(r'\[lint\] Avoid defining a one-member abstract class when a simple function will do.'), // too many false-positives; code review should catch real instances
       new RegExp(r'[0-9]+ (error|warning|hint|lint).+found\.'),
+      new RegExp(r'\[info\] TODO.+'),
```

Comment:
Hm, we should probably be basing our behavior here on whether there were any non-ignored lines. But I don't see these even when there's other warnings. What does `flutter version` say for you?